src/pages/Components/Flags/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Flags'
+order: 10
+---
+
+> A flag is used as a component header to categorize its content. Please use a [tag](/Components/Tags/) if you need to display additional information.
+
+A flag is composed of :
+
+- **A label :** A word or composed word.
+- **A background :** It can be solid or bordered, it allows to specify the degree of importance.
+
+## Variations
+
+<preview path="src/pages/Components/Flags/previews/flag-all"></preview>
+
+| Color Scheme   | Usage                                                 |
+| -------------- | ----------------------------------------------------- |
+| Primary        | For related content to Leroy Merlin                   |
```

Comment:
'For contents related to Leroy Merlin' (so everything ? can we be more precise ?)

Reply:
in a more English way : For Leroy Merlin related content
🇬🇧